bin/catkin_make_isolated

```diff
@@ -146,6 +148,7 @@ def main():
         force_cmake=opts.force_cmake,
         colorize=not opts.no_color,
         build_packages=opts.packages or ([] if opts.from_package is None else [opts.from_package]),
+        blacklisted=opts.package_blacklist[0].split(';') if opts.package_blacklist is not None else None,
```

Comment:
There is no need to split arguments by semicolon. Argparse will return a list already since the argument uses `nargs='+'`.